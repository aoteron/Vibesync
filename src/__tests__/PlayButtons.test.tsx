

describe("testing PlayButtons component", () => {
  it("snapshot", () => {
    const result = "foobar";
    expect(result).toMatchSnapshot();
  });

  test("should add two numbers", () => {
    expect(1+1).toBe(2)
  })
});
