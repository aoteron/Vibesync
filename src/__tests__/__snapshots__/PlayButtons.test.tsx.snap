// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testing PlayButtons component > snapshot 1`] = `"foobar"`;
